scheduler/connectors/services/katalogus.py

```diff
@@ -13,10 +13,37 @@ class Katalogus(HTTPService):
     def __init__(self, host: str, source: str, timeout: int = 5):
         super().__init__(host, source, timeout)
 
+        # Example:
+        #
+        # {
+        #     "organisation_id": {
+        #          "plugin-id": {}
+        #     }
+        # }
         self.organisations_plugin_cache: dict_utils.ExpiringDict = dict_utils.ExpiringDict(lifetime=30)
+
+        # Example:
+        #
+        # {
+        #     "organisation_id": {
+        #         "plugin-type": {
+        #             "plugin-id": {}
+        #         }
+        #     }
+        # }
         self.organisations_boefje_type_cache: dict_utils.ExpiringDict = dict_utils.ExpiringDict(lifetime=30)
         self.organisations_normalizer_type_cache: dict_utils.ExpiringDict = dict_utils.ExpiringDict(lifetime=30)
 
+        # Example:
+        #
+        # {
+        #     "organisation_id": {
+        #          "plugin-id": {}
+        #     }
+        # }
+        self.organisations_new_boefjes_cache: Dict = {}
```

Comment:
```suggestion
        self.organisations_new_boefjes_cache = {}
```

Reply:
Obvious assignments strictly don't need to be type hinted (unless e.g. casted from a right hand side that has multiple types using unions)

Reply:
mypy unfortunately had an opinion about it